modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -558,7 +558,40 @@ As such, ensure that `dbms.cluster.discovery.version=V2_ONLY`, and that `dbms.cl
 are set as required, so that the servers start with v2 running on the next restart.
 ====
 
+[[discovery-v1-to-v2-semi-automated-procedure]]
+=== Semi-automated procedure
 
+Another way to migrate from Discovery Service `V1` to `V2` is to use a semi-automated procedure, which simplifies the migration process
+It automates the approach described in the xref:clustering/setup/discovery.adoc#discovery-v1-to-v2-procedures[previous section].
+
+. Make sure that server side routing is enabled as detailed in xref:clustering/setup/routing.adoc#clustering-routing[Server-side routing] section
+. For all the servers, ensure that new settings are updated to the configuration as detailed the xref:clustering/setup/discovery.adoc#_preparation[Preparation] section.
+. Connect to any server using server-side routing
```

Comment:
We are not connecting 'with server side routing'. I don't believe we even need to connect with `neo4j://`, because it's not a routed query. So I would just say:
```suggestion
. Connect to any server.
```